### ISSUE TITLE
[Feature request] Mark dashboard as outdated when another user changes it

### ISSUE THREAD

**Issue by tomer-amir-vonage** [grafana/grafana#7369]
We are working with Grafana for a while now, and we are using it for our monitoring infrastructure, which means that many users are looking at the same dashboards.

As things naturally evolve and change, so do the dashboards. I know that once a user refreshes his browser he will see the latest version, but when the dashboard is presented on a big screen on live changes are being done when someone else is looking at it, there is no indication of the change.

Basically what I'm suggesting is:
1.  A small icon at the top that will indicate that someone has changed this dashboard since the last refresh.
2. An auto refresh ability for none editor users
3. For editors, a "diff view" in case they changed a dashboard at the same time as someone else?

Thanks!

**Comment by torkelo** [comment]
for TV screen you can setup a playlist for that screen and have only one dashboard in the playlist and playlist cycle time of 1h hour  (or smaller), and the dashboard will be completely reloaded every hour, you also have the options  to then change what dashboard is played on that TV remotely by changing the playlist  :)

**Comment by daniellee** [comment]
This has been implemented: #27583

It is behind a feature toggle but will go live in Grafana 7.4.

https://grafana.com/blog/2020/10/29/grafana-7.3-released-support-for-the-grafana-tempo-tracing-system-new-color-palettes-live-updates-for-dashboard-viewers-and-more/#live-updates-broadcast-to-dashboards
